packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceAuthWebLogin.test.ts

```diff
@@ -227,3 +228,40 @@ describe('Command chosen is based on results of isDemoMode()', () => {
     expect(createExecutor() instanceof ForceAuthWebLoginExecutor).to.be.true;
   });
 });
+
+describe('Should work correctly in container mode', () => {
+  afterEach(() => {
+    delete process.env.SFDX_CONTAINER_MODE;
+  });
+  it('and use force:auth:web:login when container mode is not defined', () => {
+    const authWebLogin = new ForceAuthWebLoginExecutor();
+    const authWebLoginCommand = authWebLogin.build(
+      ({} as unknown) as AuthParams
+    );
+    expect(authWebLoginCommand.toCommand()).to.equal(
+      'sfdx force:auth:web:login --setalias  --instanceurl  --setdefaultusername'
+    );
+  });
+
+  it('and use force:auth:web:login when container mode is empty', () => {
```

Comment:
I don't think we use `and` to denote the `it(...)`.

I'm open to discuss this but it's better to follow the established convention of a file.